kore/test/Test/Kore/Reachability/Prove.hs

```diff
@@ -402,28 +211,7 @@ test_reachabilityVerification =
             ]
             [ simpleAllPathClaim Mock.a (mkOr Mock.b Mock.c) ]
             []
-        assertEqual ""
-            (Right ())
-            actual
-    , testCase "Mixed: Concurrent rules" $ do
-        -- Axioms:
-        --     a => b
-        --     a => c
-        -- Claims: a => b #Or c
-        -- Expected: success
-        actual <- proveClaims_
-            Unlimited
-            (Limit 2)
-            [ simpleAxiom Mock.a Mock.b
-            , simpleAxiom Mock.a Mock.c
-            ]
-            [ simpleOnePathClaim Mock.a (mkOr Mock.b Mock.c)
-            , simpleOnePathClaim Mock.a (mkOr Mock.b Mock.c)
-            ]
-            []
-        assertEqual ""
-            (Right ())
-            actual
+        assertEqual "" MultiAnd.top actual
```

Comment:
Could these tests be combined like the tests above were?

Reply:
Yes, I cleaned up the test suite some more.